guillotina/contrib/dbusers/content/users.py

```diff
@@ -55,12 +52,14 @@ class IUser(IFolder, IPrincipal):
     behaviors=["guillotina.behaviors.dublincore.IDublinCore"],
 )
 class User(Folder):
-    username = email = name = password = None
-    disabled = False
-    user_roles = ["guillotina.Member"]
-    user_groups: typing.List[str] = []
-    user_permissions: typing.List[str] = []
-    properties: typing.Dict[str, typing.Any] = {}
+    def __init__(self, *args, **kwargs):
+        self.user_groups = []
```

Comment:
```suggestion
        self.user_groups: typing.List[str] = []
```

Reply:
and the same for the others

Reply:
What's tha point on typing this if types doesn't esxist at runtime?

Reply:
But also, we removed them, because it's already typed with the schema definition

Reply:
On the end, interfaces are validated at runtime.. mypy types not. doesn't make sense to double type something.